kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -267,7 +267,10 @@ define([
             }).then(() => {
                 this.renderTable(jobs);
                 this.setUpEventHandlers();
-                this.setUpJobListeners(jobs);
+                this.jobManager.bus.emit(jcm.MESSAGE_TYPE.STATUS, {
```

Comment:
No longer need to set up listeners as the job manager does that. Use the JM shortcut for requesting batch info and request a batch update at the same time.